### ISSUE TITLE
'Uncaught ReferenceError: process is not defined' when not using npm

### ISSUE THREAD

**Issue by geoffreycastillo** [taye/interact.js#890]
I use interact.js from jsDelivr as such:

`<script src="https://cdn.jsdelivr.net/npm/interactjs/dist/interact.min.js"></script>`

I don't use npm and it was working fine.

Now I get the error:

`Uncaught ReferenceError: process is not defined`

It seems this error is fairly recent. I've tested different versions and it seems it was introduced in 1.10.6. Rolling back to 1.10.5 with

`<script src="https://cdn.jsdelivr.net/npm/interactjs@1.10.5/dist/interact.min.js"></script>`

solves the issue

Seems the problem is coming from line 101 of InteractStatic.ts:

`interact.version = process.env.npm_package_version`

**Comment by robdplatt** [comment]
We've recently started having the same issue as @geoffreycastillo

**Comment by mitchjacksontech** [comment]
Same issue here.  Pin your CDN include to the previous release version for immediate fix:
```html
<script src="https://cdn.jsdelivr.net/npm/interactjs@1.10.3/dist/interact.min.js"></script>
```

The stack-trace I see from this error:
```
caught ReferenceError: process is not defined
    createInteractStatic InteractStatic.ts:101
    t scope.ts:83
    <anonymous> index.ts:3
    <anonymous> _prelude:1
    <anonymous> _prelude:1
```

**Comment by pegues** [comment]
Also experiencing this issue.

**Comment by Zeffuro** [comment]
Same issue going back to previous version for now.

**Comment by taye** [comment]
Should be fixed now for `v1.10.7`
